core/src/main/java/com/valtech/aem/saas/core/fulltextsearch/DefaultFulltextSearchService.java

```diff
@@ -171,6 +171,7 @@ private ResultDTO getResult(SearchResultDTO searchResultDto, HighlightingDTO hig
                         .description(new HighlightedDescriptionResolver(searchResultDto,
                                                                         highlightingDto).getDescription())
                         .bestBet(searchResultDto.isElevated())
+                        .repositoryPath(searchResultDto.getRepositoryPathUrl())
```

Comment:
this is optionally, right?

Reply:
it could be null, if not present in the response from saas api